README.md

```diff
@@ -1 +1,25 @@
-# couch-tracker-documentation
\ No newline at end of file
+# Couch-Tracker
+
+An Android application, to keep track of the TV shows you follow, and the movies you've watched.
+
+This repository contains all the documents and design decisions we've made.
+
+
+## Tenets
+
+- Open-source
+- no ads, no trackers and no third party integrations which are not strictly necessary to make the app work
+- no paywalling for features done locally and don't require an ongoing cost from us
+- Low (or zero) licensing/hosting fees, low architectural complexity
+- User data is owned by each user, and is always available to them. This data is in an open format
+- Focused on user data: discover/explore sections and the likes may have a place, but it is not central
+- Fast and simple
```

Comment:
I think we can put this as a second point

Reply:
ok